definitions/npm/cache-manager_v2.x.x/flow_>=v0.28.x/cache-manager_v2.x.x.js

```diff
@@ -0,0 +1,24 @@
+declare module 'cache-manager' {
+  declare type _CacheOptions = {
+    ttl?: number,
```

Comment:
Thanks, This was copied straight from out private work libdefs.
Wasn't surprised something was missing.

Added options & wrap function.